src/app/components/quotes/quotes.hbs

```diff
@@ -1,36 +1,36 @@
 <style scoped>
-    @import url(/app/pages/home/quotes/quotes.css);
+    @import url(/app/components/quotes/quotes.css);
 </style>
 <div class="quotes">
     <div class="quote-bucket right">
-        <img src="/images/home/quotes/juan.jpg" width="175" height="230" />
+        <div class="image juan"></div>
```

Comment:
It may get clearer when you're actually getting info from the back end. I think the server ought to deliver something named more like "first-image" than "juan". And your stuff would be similar. I'm done reviewing.